test-openjdk/src/test/java/com/redhat/qe/openjdk/OpenJDKTestParent.java

```diff
@@ -133,7 +133,7 @@ public static Build startBinaryBuild(final String bcName, Path sources) throws I
 			ExecutorService executorService = Executors.newSingleThreadExecutor();
 			final Future<?> future = executorService.submit(() -> {
 				Collection<File> filesToArchive = FileUtils.listFiles(sources.toFile(), TrueFileFilter.INSTANCE, TrueFileFilter.INSTANCE);
-				try (TarArchiveOutputStream o = (TarArchiveOutputStream) new ArchiveStreamFactory().createArchiveOutputStream(ArchiveStreamFactory.TAR, pos)) {
+				try (TarArchiveOutputStream o = new ArchiveStreamFactory().createArchiveOutputStream(ArchiveStreamFactory.TAR, pos)) {
```

Comment:
This looks like deja-vu. I think we have already did  TarArchiveOutputStream removal/addition or change f type, or maybe we did similar operation else where. I s it actually related to change or just ide highlighted redundant retype? Or was the retype need forced by update?

Reply:
It was the ide flagging the redundancy.